x-pack/plugins/apm/server/routes/errors/get_error_groups/get_error_group_main_statistics.ts

```diff
@@ -128,16 +135,17 @@ export async function getErrorGroupMainStatistics({
   );
 
   return (
-    response.aggregations?.error_groups.buckets.map((bucket) => ({
-      groupId: bucket.key as string,
-      name: getErrorName(bucket.sample.hits.hits[0]._source),
-      lastSeen: new Date(
-        bucket.sample.hits.hits[0]?._source['@timestamp']
-      ).getTime(),
-      occurrences: bucket.doc_count,
-      culprit: bucket.sample.hits.hits[0]?._source.error.culprit,
-      handled: bucket.sample.hits.hits[0]?._source.error.exception?.[0].handled,
-      type: bucket.sample.hits.hits[0]?._source.error.exception?.[0].type,
-    })) ?? []
+    response.aggregations?.error_groups.buckets.map((bucket) => {
+      const source = bucket.sample.hits.hits[0]._source as APMError;
```

Comment:
You sure you need to cast it, i am getting on my id the type properly

![image](https://github.com/elastic/kibana/assets/7416358/7c73aa91-8c40-4897-8ef3-7b2aba12251d)

Reply:
Did you try with the change to sources?

Reply:
<img width="1185" alt="image" src="https://github.com/elastic/kibana/assets/31922082/d156e883-1655-4d90-aad1-5fd783ca72e6">

Reply:
ah now i see, the changes to sources is causing it 👍🏼 
my bad, didn't see that, after i checkout out your branch i see it now